test/suites/standard/networking-smoke.robot

```diff
@@ -67,3 +69,22 @@ Create Hello MicroShift Ingress
 Delete Hello MicroShift Ingress
     [Documentation]    Delete ingress for cleanup.
     Oc Delete    -f ${HELLO_USHIFT_INGRESS} -n ${NAMESPACE}
+
+Resolve Hello MicroShift MDNS
```

Comment:
I think I would have called this something like "DNS entry should resolve" and passed the name as an argument. Let's take this version, and I can propose an update and see if you like it.

Reply:
See #2386

Reply:
I'm fine with the change :)

Reply:
Included them in this PR